packages/react/src/Bento/Bento.tsx

```diff
@@ -0,0 +1,298 @@
+import React, {ReactHTML, ReactElement, forwardRef, useCallback, useMemo, Ref, PropsWithChildren} from 'react'
+import clsx from 'clsx'
+import {Icon, IconProps} from '@primer/octicons-react'
+import {useWindowSize, BreakpointSize} from '../hooks/useWindowSize'
+import type {BaseProps} from '../component-helpers'
+import {Heading, Text, Link, HeadingProps, TextProps, LinkProps, ColorModesEnum} from '../'
+
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/bento/colors-with-modes.css'
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/bento/base.css'
+import styles from './Bento.module.css'
+
+export type Size = 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge'
+export type ColumnIndex = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12
+
+type ResponsiveColumnIndex = Partial<Record<Size, ColumnIndex>>
+type ResponsiveRowIndex = Partial<Record<Size, number>>
+
+type Flow = 'row' | 'column'
+type ResponsiveFlow = Partial<Record<Size, Flow>>
+
+type Align = 'start' | 'center' | 'end'
+type ResponsiveAlign = Partial<Record<Size, Align>>
+
+type ReducedAlign = 'start' | 'center'
+type ResponsiveReducedAlign = Partial<Record<Size, Align>>
```

Comment:
The value is the original align, should it be the subset instead?

```suggestion
type ResponsiveReducedAlign = Partial<Record<Size, ReducedAlign>>
```